RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/dialogs/ProjectInfoDialog.java

```diff
@@ -77,9 +84,9 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         projectTitle.setText(book + " (" + bookCode + ")");
         languageTitle.setText(language + " (" + languageCode + ")");
         translator.setText(translators);
-        if(translation.compareTo("ulb") == 0) {
+        if (translation.compareTo("ulb") == 0) {
```

Comment:
I've always wondered about this. Even though I understand what it does, my brain always takes slightly more effort to interpret this. What is the reason for using `compareTo()` as opposed to `equals()`?

For example:
`if (translation.compareTo("ulb") == 0) {}`
vs.
`if (translation.equals("ulb")) {}`